packages/dashboard/src/portal/components/TftSwapPrice.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <v-container>
+    <div class="d-flex ml-5">
+      <div class="d-flex" style="align-items: center">
+        <p>{{ prices[0].amount }} {{ prices[0].currency }}</p>
+        <v-tooltip>
+          <template v-slot:activator="{ on, attrs }">
+            <v-btn @click="priceSwap" icon v-bind="attrs" v-on="on" class="d-flex align-center">
+              <v-icon>mdi-swap-horizontal</v-icon>
+            </v-btn>
+          </template>
+          <span>ThreeFold Price Swapping</span>
+        </v-tooltip>
+        <p>{{ prices[1].amount }} {{ prices[1].currency }}</p>
+      </div>
+    </div>
+  </v-container>
+</template>
+<script lang="ts">
+import { Component, Vue } from "vue-property-decorator";
+
+type SwapPrice = {
+  currency: string;
+  amount: number;
+};
+
+@Component({
+  name: "TftSwapPrice",
+})
+export default class FundsCard extends Vue {
```

Comment:
Why is it called FundsCard? isn't that TFTPriceCard?

Reply:
updated